src/networking/APNSNetworkService.php

```diff
@@ -0,0 +1,119 @@
+<?php
+declare( strict_types = 1 );
+
+class APNSNetworkService {
+
+	/** @var resource **/
+	private $curl_handle;
+
+	/** @var int */
+	public $port = 443;
+
+	/** @var bool **/
+	private $debug = false;
+
+	/** @var bool **/
+	private $ssl_verification_enabled = true;
+
+	public function __construct() {
+		$ch = curl_multi_init();
+		curl_multi_setopt( $ch, CURLMOPT_PIPELINING, CURLPIPE_MULTIPLEX );
+		curl_multi_setopt( $ch, CURLMOPT_MAX_TOTAL_CONNECTIONS, 1 );
+		curl_multi_setopt( $ch, CURLOPT_PIPEWAIT, 1 );
```

Comment:
@jkmassel the code in `index.php` seems to work with this option, but I get this warning

```
PHP Warning:  curl_multi_setopt(): Invalid curl multi configuration option in /Users/gio/Developer/a8c/php-push/src/networking/APNSNetworkService.php on line 22
```

I haven't been able to test a successful notification delivery using `index.php`, though. I can  replicate a failed one using the wrong token for the WordPress Development app, but I can't get my token from MC anymore.

![image](https://user-images.githubusercontent.com/1218433/89390801-9aa0c280-d74a-11ea-9bcb-094442a5059b.png)

Reply:
Hmm ok – it might be missing on your local PHP copy 🤔 

We can push it to a future PR I think just to make things easy, but could we leave a TODO here so we don't forget?